### ISSUE TITLE
8.13 has been released

### ISSUE THREAD

**Issue by lenovouser** [sameersbn/docker-gitlab#938]
[gitlab.com/blog/gitlab-8-13-released](https://about.gitlab.com/2016/10/22/gitlab-8-13-released/)

/cc @sameersbn @solidnerd

**Comment by reynold-lariza** [comment]
anyone tested 8.13.0 yet?

**Comment by solidnerd** [comment]
I need some time for it. I hope i can make it today or tomorrow. Otherwise I would appreciate a WIP PR for the next update.

**Comment by reynold-lariza** [comment]
**Environment/observation with 8.13.0**

My testing is done with an existing (external) data for git, database, and redis. I had to revert to 8.12.7 yesterday morning, after I found that when I tried to accept a MR from one branch to another branch to an existing project. 

**Behavior:**
The accept merge request button get stuck, and nothing happens. Then after refreshing the page, the MR wasn't merged.
